config/initializers/delayed_job_config.rb

```diff
@@ -0,0 +1,7 @@
+Delayed::Worker.logger ||= if defined?(Rails)
+                             Rails.logger
+                           elsif defined?(RAILS_DEFAULT_LOGGER)
```

Comment:
Out of curiosity, why this many loggers?

Reply:
To be honest, I am not 100% sure, this was boilerplate I took from registers-frontend.